doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,85 @@
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section is intended for users that would like to re-distribute the |NCS|
+with additions or changes to it.  This can be useful if you are a software or
+hardware vendor that would like to augment the |NCS| with additional
+functionality, while at the same time still benfitting from the infrastructure
+it provides.
+
+The contents of this section are not comprehensive, they are intended as
+suggestions for those wanting to extend the |NCS| but other mechanisms may be
+used as long as they comply with the text in the :file:`LICENSE` file that you
+can find at the root of every Git repository in the |NCS|. In case of doubt
+please reach out to Nordic Semiconductor.
+
+Getting added to the main manifest
+**********************************
+
+The simplest way (for your users or customers) to become part of the |NCS| is to
+be included in its official `west manifest file`_. In this way your own
+:ref:`module repository <zephyr:modules>` will automatically be part of the
+|NCS| official distribution and will be included in all releases.
+
+This solution is reserved for selected vendors and requires a commercial
+agreement with Nordic Semiconductor. Please contact your local representative if
+you'd like to be considered for inclusion.
+
+Some examples of this type of integration are:
+
+* :ref:`Memfault <mod_memfault>`. This functionality is present via the addition
+  of a `west project in the |NCS| manifest <Memfault entry in the manifest>`_.
+  The `repository provided by Memfault <Memfault firmware SDK>`_ is public,
+  and so |NCS| users will obtain a copy of it (as well as the integration code
+  necessary to be used in |NCS|-based aplicaitons) by default when :ref:`getting
+  the code <cloning_the_repositories>`.
+
+* ANT protocol support. In this particular case the code and documentation of
+  this protocol is confidential, and in fact access to the private module repository
+  requires the GitHub user to be added to it. This is why the west project is
+  disabled by default, using the :ref:`project groups
+  <zephyr:west-manifest-groups>` feature of west. This means that the ANT entry
+  in the manifest is disabled by default via its presence in the `group
+  filter <manifest group filter>`_. Users will not get a copy of this repository
+  when :ref:`getting the code <cloning_the_repositories>`, and will instead need
+  to enable the repo first using west itself to fetch it locally::
+
+    west config manifest.group-filter +ant
+    west update
+
+Providing a Zephyr module
+*************************
+
+This approach is essentially identical to the previous one, but with the
+difference that the :ref:`module repository <zephyr:modules>` you create will
+not be part of the |NCS| official distribution by default. Instead, you will
+need to host your own documentation explaining users how to add the module you
+make available to their :ref:`own manifest <dm_workflow_4>`.
```

Comment:
Optional, but we have documentation as an example in our [README](https://github.com/golioth/golioth-zephyr-sdk#using-with-nrf-connect-sdk) and [documentation site](https://docs.golioth.io/hardware/nrf91/zephyr-quickstart/set-up-zephyr).

Reply:
Added, thanks!